.github/workflows/release.yml

```diff
@@ -85,14 +88,18 @@ jobs:
           cosign-release: v2.0.0
 
       - name: Install Rust toolchain
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: 1.68
-          default: true
-          target: ${{ matrix.config.target }}
+        shell: bash
+        run: |
+          rustup toolchain install ${{ env.RUST_VERSION }}
```

Comment:
for my own curiosity: how do you have access to the rustup binary here? does it come standard with the os?

Reply:
Yes, Rust tools like rustup and cargo come pre-installed in the GitHub runners.